joomla/joomla-cms
# TranslateFormat in all other forms

This PR extends the use of the new feature for translated date (and time) formats in the calendar formfield  to all other core extensions.
https://github.com/joomla/joomla-cms/pull/12102 originally introduced that feature but only did it for com_content.

### Summary of Changes
Replace the hardcoded format in the form declaration with `translateFormat="true"` and `showTime="true"` where appropriate.

### Testing Instructions
* Add the new language strings to your second language you have installed. Eg for de-DE add this to both de-DE.ini files (frontend and backend):
```
DATE_FORMAT_CALENDAR_DATE="%d.%m.%Y"
DATE_FORMAT_CALENDAR_DATETIME="%d.%m.%Y %H:%M:%S"
DATE_FORMAT_FILTER_DATETIME="d.m.Y H:i:s"
DATE_FORMAT_FILTER_DATE="d.m.Y"
```
* Make sure all instances of a calendar formfield (usually the created/modified/publish_up/publish_down) are using the translated format and don't create an error when saving.

### Documentation Changes Required
None that I'm aware of.
